PiwikTracker/PiwikTracker.swift

```diff
@@ -260,13 +216,15 @@ extension PiwikTracker {
     ///
     /// - Parameter view: An array of hierarchical screen names.
     /// - Parameter url: The url of the page that was viewed. If none set the url will be http://example.com appended by the screen segments. Example: http://example.com/players/john-appleseed
-    @objc public func track(view: [String], url: URL? = nil) {
-        queue(event: event(action: view, url: url))
+    public func track(view: [String], url: URL? = nil, dimensions: [CustomDimension] = []) {
```

Comment:
As `CustomDimension` isn't visible to objc, I'm not sure if these methods are still visible to objc.

Reply:
There is a Objective-C extension without the dimensions property. See https://github.com/piwik/piwik-sdk-ios/pull/193/files#diff-bd7b3c5478dc6552eb40fd9baaa55d18R289